pkg/server/status.go

```diff
@@ -3055,10 +3053,6 @@ func (s *systemStatusServer) localHotRanges(
 		return nil, err
 	}
 
-	// sort the slices by cpu
```

Comment:
why do we no longer need to sort?

Reply:
We do, removing this is a mistake. Adding it back.